Sources/HaCWebsiteLib/ViewModels/LandingPage.swift

```diff
@@ -4,6 +4,12 @@ import HaCTML
 
 struct LandingPage {
   let updates: [PostCard]
+  let youtube_url = "https://www.youtube.com/hackersatcambridge"
```

Comment:
Nit: we're camelCase here (so `youtubeURL`)

Reply:
Fixing now...

Reply:
foxed @_